tool/timer.re

```diff
@@ -17,17 +15,13 @@ impl TimerState {
         let ls = Tas::get_level_state();
         ls.end_seconds.to_float() + ls.end_partial_seconds
     }
```

Comment:
`get_start_time` and `get_end_time` don't actually access `TIMER_STATE` i.e. `self` and thus don't need it as an argument. They don't need to be methods, they can be static functions instead and be called like `TimerState::get_{start,end}_time`.